diesel/src/pg/expression/functions.rs

```diff
@@ -784,18 +784,18 @@ define_sql_function! {
     /// #     Ok(())
     /// # }
     /// ```
-    fn array_to_string_null<Arr: ArrayOrNullableArray<Inner=Elem> + SingleValue, Elem: SingleValue>(
+    #[sql_name = "array_to_string"]
+    fn array_to_string_with_null_string<Arr: ArrayOrNullableArray + SingleValue>(
         arr: Arr,
         delim: Text,
-        null_str: Nullable<Text>
+        null_str: Text
     ) -> Text;
 }
 
 #[cfg(feature = "postgres_backend")]
 define_sql_function! {
-    #[sql_name = "array_to_string"]
     /// Converts each array element to its text representation, and concatenates those separated by the delimiter string.
-    /// This variant omits the `null_str` argument.
+    /// This variant omits the `null_str` argument. See [array_to_string_null] for a variant with that argument
```

Comment:
As you rename this function, new name should be used:
```suggestion
    /// This variant omits the `null_str` argument. See [`array_to_string_with_null_string`] for a variant with that argument
```